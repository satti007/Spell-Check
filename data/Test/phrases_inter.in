from the eath to the moon
wate fountain at school
roff of the house
a geant leap for mankind
interested in your opinin
hope for the futre
taking an extreem step
walking down the aisel
cops and robers
presient of united states
a few goode men
a briht sunny day
raining cats and doggs
chill down the spene
broken hart
keep out of reech
I cannot thank you enugf
keep your fingers krossd
read bitwin the lines
sky fool of stars
follow the skedule
looosing the plot
hiway to the city
peace of cake
burn the midnite oil
wonder the chair		
after this own		
hind the sofa
oad the table
I dont udrstnd		
To complete the moth problem
moving flow flower to flower
crying the math problem
sevnten hungry lions in the rocks
A fate worse than dead
A golden key can open any doer
A mad after my own heart
a calm sea does not make a skied sailor
constant occpton prevents temptation
a dangerous foreseen is half avoided
early to bed and curly to rise makes a man healthy wealthy and wise
Experience is the father from wisdom
facts speak blouder than words
frndsip is love with understanding
put all your eggs in one blanket
a rotten apple sols the barrel
