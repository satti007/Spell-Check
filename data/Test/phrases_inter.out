eath	earth
wate	water
roff	roof
geant	giant
opinin  opinion
futre	future
extreem	extreme
aisel	aisle
robers	robbers
presient	president
goode	good
briht	bright
doggs	dogs
spene	spine
hart	heart
reech   reach
enugf   enough
krossd  crossed
bitwin  between
fool    full
skedule schedule
looosing        losing
hiway   highway
peace   piece
midnite midnight
wonder	under
own	one
hind	behind
oad	over
udrstnd	understand		
moth	math
flow	from
crying	trying
sevnten	seventeen
dead	death
doer	door
mad	man
skied	skilled
occpton	occupation
dangerous	danger
curly	early
from	of
blouder	louder
frndsip	friendship
blanket	basket
sols	spoils
